pkg/server/status/runtime.go

```diff
@@ -551,8 +551,6 @@ func subtractDiskCounters(from *diskStats, sub diskStats) {
 	from.readCount -= sub.readCount
 	from.readTimeMs -= sub.readTimeMs
 	from.readBytes -= sub.readBytes
-
-	from.iopsInProgress -= sub.iopsInProgress
```

Comment:
I'd be inclined to rename the method now, but I can't think of a great name immediately.

Reply:
yeah idk. NormalizeWithRespectToStartingValues or something

Reply:
tare :balance_scale: